api/lib/infrastructure/repositories/campaign-collective-result-repository.js

```diff
@@ -142,3 +182,26 @@ function _forgeCampaignCompetenceCollectiveResults(campaignId, competences, part
     .value();
 }
 
+function _forgeCampaignTubeCollectiveResults(campaignId, tubes, competences, participantCount, targetedSkillsByTubeId, participantsKECountBySkillId) {
+  return _(targetedSkillsByTubeId).keys().map((tubeId) => {
+    const tube = _.find(tubes, { id: tubeId });
+
+    const competenceId = targetedSkillsByTubeId[tubeId][0].competenceId;
+    const competence = _.find(competences, { id: competenceId });
+
+    const averageValidatedSkills = targetedSkillsByTubeId[tubeId].reduce((result, skill) =>
+      participantsKECountBySkillId[skill.id] ? result + participantsKECountBySkillId[skill.id] : result, 0) / participantCount || 0;
```

Comment:
Encore un `reduce` qui tape l'incruste 😄 
```suggestion
    const averageValidatedSkills = _.sumBy(skills, (skill) => participantsKECountBySkillId[skill.id] || 0) / participantCount || 0;
```

Reply:
J'ai remplacé la reduce par le `sumby` :)